### ISSUE TITLE
Documentation for S(Q) models need updating (Trac #1148)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#1189]
The model documentations auto generates some of the documentation such as the example plot (using default parameters) and adds the scale and background parameters to the table.

For S(Q) however there is no scale and background term when using the model (a change from previous versions I believe) but the documentation still generates those leading to some user confusion.

Migrated from http://trac.sasview.org/ticket/1148
```json
{
    "status": "closed",
    "changetime": "2019-03-03T15:24:25",
    "_ts": "2019-03-03 15:24:25.638383+00:00",
    "description": "The model documentations auto generates some of the documentation such as the example plot (using default parameters) and adds the scale and background parameters to the table.\n\nFor S(Q) however there is no scale and background term when using the model (a change from previous versions I believe) but the documentation still generates those leading to some user confusion.",
    "reporter": "butler",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Documentation",
    "time": "2018-07-25T17:43:55",
    "component": "SasView",
    "summary": "Documentation for S(Q) models need updating",
    "priority": "critical",
    "keywords": "",
    "milestone": "SasView 4.3.0",
    "owner": "butler",
    "type": "enhancement"
}
```

**Comment by butlerpd** [comment]
Trac update at `2018/08/07 14:09:52`: 
* **butler** changed owner from "" to "butler"
* **butler** changed status from "new" to "assigned"

**Comment by pkienzle** [comment]
Trac update at `2018/08/17 15:53:57`: **pkienzle** commented:

Fixed in sasmodels.  Close ticket when verified.

**Comment by butlerpd** [comment]
Trac update at `2018/08/19 18:21:00`: 
* **butler** commented:

Fixed by commit!6e45516f19370dc8d3a8ff3fa00959f6e27d9e19. Tested that it does as advertized. Was pushed directly to master rather than through a PR as was a rather simple fix to documentation and sasmodels cannot yet have builds for testing. Basically create a conditional block in generate.py so that "COMMON" parameters (i.e. those auto generated for every mode) are not added to the parameter table in the generated docs for structure factor models (but is for all others). 

Now closing ticket


* **butler** changed resolution from "" to "fixed"
* **butler** changed status from "assigned" to "closed"

**Comment by smk78** [comment]
Trac update at `2018/10/02 10:40:34`: 
* **smk78** commented:

Re-opening this ticket as I just noticed that none of the S(Q) models actually describe the functions they are applying!


* **smk78** changed milestone from "SasView 4.2.0" to "SasView 4.3.0"
* **smk78** changed resolution from "fixed" to ""
* **smk78** changed status from "closed" to "reopened"
* **smk78** changed type from "defect" to "enhancement"
* **smk78** changed workpackage from "SasView Bug Fixing" to "SasView Documentation"

**Comment by pkienzle** [comment]
Trac update at `2018/11/15 15:56:40`: **pkienzle** commented:

The ticket description and possible the title need to be changed to reflect this new requirement.

**Comment by butlerpd** [comment]
Trac update at `2019/03/03 15:24:25`: 
* **butler** commented:

Indeed this ticket was not about the lack of equation or other lack in the writeup but about the autogeneration doing something wrong.  That was fixed in the commit listed.  The title was probably not the best.

Going to reclose this ticket and open a new one as actually this is **not** a problem specific to SQ. I have found there are several models that do not have the equations listed ...



* **butler** changed resolution from "" to "fixed"
* **butler** changed status from "reopened" to "closed"
